src/value.rs

```diff
@@ -0,0 +1,179 @@
+use std::fmt;
+
+use bevy_console_parser::{Value, ValueRawOwned};
+
+/// The value types available for arguments.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum ValueType {
+    /// String arg type
+    String,
+    /// Int arg type
```

Comment:
What type of integer are these stored as?

Reply:
Int is stored as `i64`, and Float is stored as `f64`
https://github.com/RichoDemus/bevy-console/pull/4/files/b2d9dfd423fbf978c03794f00f4d0ac3f954f977#diff-df3933aef2020e10367701a3160b4c9efc31e32d7202019c8a17de0f267da116R20-R21

You can define a command with
 - `u8` up to `u128` (including `usize`)
 - `i8` up to `i128` (including `isize`)
 - `f32` and `f641
 
But these values will be casted using `as`. Maybe I should remove `128` bit types, and any types that could result in a potentially lossy cast.
https://github.com/RichoDemus/bevy-console/pull/4/files/b2d9dfd423fbf978c03794f00f4d0ac3f954f977#diff-be1c878c753064b7eaddf4a10ba4215a93dd1817d31e6788691cdb29752fd612R117

Reply:
Good; I think these are sizable defaults. I agree with removing 128 bit types: they're too niche and will fail surprisingly. This comment should be documented on those fields :)

Reply:
Honestly, i64 can only be safely casted as i128. I think I should remove the implementations and only implement `i64` and `f64` to avoid any unsafe casting.
If the user wants to cast, they can put i64/f64 in their struct and cast manually I think.

So this means the user cannot put `u8` and such in their command arguments